docs/section23/Rule23-1.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 23-1  
+
+**Rule ID:** 23-1  
+**Rule Description:** For baseline systems 5-8 and 11, the SAT is reset higher by 5F under minimum cooling load conditions.  
+**Rule Assertion:** B-RMR = expected value  
```

Comment:
@mkarpman it seems strange that App G has this SAT reset requirement for system 11. System 11 is a single zone system, the SAT will automatically be based on the heating or cooling demand of that zone, correct? Does SAT reset even apply to such a system in reality?

Reply:
I think the intent is that it applies to all VAV systems since they can adjust amount of extracted heat by adjusting either flow rate or SAT. So it is my understanding that this requirement say that SAT can go up from 55F to 60F, and after that the further reduction in heat extraction would be due to reduced flow rate.

Reply:
got it, this is helpful.